### DISCUSSION TITLE
built in support for mapping plugins like dynmap

### DISCUSSION THREAD

**Discussion by geekmoto13** [pelican-dev/panel#176] [Ideas]
it would be nice if all i had to do was tick a box or select an item in a drop down to enable plugins like dynmap so I dont have to ssh in and mess with ubuntu firewall to make that work

**Comment by Boy132** [comment]
Not sure what you are on about. You just need to add the Dynmap port as allocation to the server and it will work.

**Reply by geekmoto13** [reply]
one click port allocation, I.E. >click HERE < to allocate port for dynmap or it can ask which port to use if user changed the port in the config

**Reply by realkarmakun** [reply]
What? Why? That's so specific there is no reason to add that. Perhaps something like could be made through Plugin system. It would depend on API available to plugins though

**Reply by geekmoto13** [reply]
because not every one knows to do that, and also it would just be a nice to have

**Comment by lancepioch** [comment]
This can be made into a plugin, thanks!
